src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
-
+server.port=9090
```

Comment:
Why requires to update the web application port.

Reply:
Because docker runs on 8080

Reply:
Docker is not a service, the container itself doesn't occupy any port, only the service which is running on the container.